README.md

```diff
@@ -1,3 +1,45 @@
-# resin.io docs
+Resin Documentation
+---------
```

Comment:
Ugh, there are actually some projects that still use dashes. I'll make a note to update myself!